README.md

```diff
@@ -26,6 +32,13 @@ Verify that the Sass can compile by running
 $ npx sass ./test/main.scss --load-path=./bower_components/
 ```
 
+For npm install only:
+Verify that the Sass can compile by running
```

Comment:
**Suggestion:** Could this follow on from the previous could block to say

```suggestion
And by running:
```

Reply:
@joelcarr thank you, I have updated it now!